Control.h

```diff
@@ -0,0 +1,44 @@
+﻿/* Roborregos 2020.
+ * This Control class has all the fuctions to get
+ * The angle, desired angle and sensors error.
+ * This class is so necessary to move the robot to
+ * The correct direction of the robot.
+*/
+#ifndef CONTROL_H
+#define CONTROL_H
+
+#include <Wire.h>
+#include "arduino.h"
+#include "Calibration.h"
+
+class Control {
+  
+  public:
+    Control();
+    Adafruit_BNO055 bno; 
+
+    // Get the difference of the current angle with the 0, and return a new error.
+    double getDifferenceWithZero();
+    // Get the current angle, return the event.orientation.x through a function.
+    double getAngleBNOX();
+    // Get the current angle, return the event.orientation.Y through a function.
+    double getAngleBNOY();
+    // Get the current angle, return the event.orientation.Z through a function.
+    double getAngleBNOZ();
```

Comment:
Maybe all these should be on their own BNO.h and BNO.cpp

Also, on these files you can add the calibration functions you have on Calibration.h and Calibration.cpp